kernel/dune

```diff
@@ -3,8 +3,8 @@
  (synopsis "The Coq Kernel")
  (public_name coq-core.kernel)
  (wrapped false)
- (modules (:standard \ genOpcodeFiles uint63_31 uint63_63 float64_31 float64_63))
- (libraries boot lib coqrun dynlink))
+ (modules (:standard \ genOpcodeFiles))
+ (libraries boot lib coqrun dynlink primitive))
```

Comment:
@proux01 This is where "coq-core" depends on "coq-primitive".

Reply:
That doesn't seem enough according to the CI.

Reply:
Right, in the dune-project file the package for coq-core also needs to depend on coq-primitive. Look at how coq-stdlib relies on coq-core.